### PR TITLE
Past code review

### PR DESCRIPTION
**Pull request by Kaghenimbale**
- [ ]  Create a new empty repo.
- [ ]  Copy the code from the project that you completed during the Admissions Trial
- [ ]  Create a new branch.
- [ ] Open a pull request.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+      <input type="text" />
+      <button type="button" title="getstarted">Get Started</button>
+    </section>
+
+    <div class="section-2">
+      <div class="list-item">
+        <div class="list-icon">
+          <img
+            src="image/fire-sign-fire-flame-icon-isolated-on-white-background-illustration-free-vector.webp"
+            alt=""
```
Comment (Kaghenimbale) [inline index.html:47]: - [ ] Great job adding the icon image :+1:   However you could also add text in the alt attribute of the img tag, this might give to the user an idea of the image type you are displaying when the image is loading and for [accessibility](https://developer.mozilla.org/en-US/docs/Web/Accessibility) reason. Kindly address this. :smile:

File: index.html [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+            will increase the longevity of your purchase.
+          </p>
+        </div>
+      </div>
+
+      <div class="list-item">
+        <div class="list-icon">
+          <img
+            src="image/car-icon-in-flat-style-simple-traffic-icon-free-vector.webp"
+            alt=""
```
Comment (Kaghenimbale) [inline index.html:63]: - [ ] Great job adding the icon image :+1:   However as suggested above :upside_down_face: you could also add text in the alt attribute of the img tag, this might give to the user an idea of the image type you are displaying when the image is loading and for [accessibility](https://developer.mozilla.org/en-US/docs/Web/Accessibility) reason. Kindly address this. :smile:

File: index.html [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+            will increase the longevity of your purchase.
+          </p>
+        </div>
+      </div>
+
+      <div class="list-item">
+        <div class="list-icon">
+          <img
+            src="image/battery-icon-260nw-432206269.webp"
+            alt=""
```
Comment (Kaghenimbale) [inline index.html:79]: - [ ] Great job adding the icon image :+1:   However as suggested above :upside_down_face: you could also add text in the alt attribute of the img tag, this might give to the user an idea of the image type you are displaying when the image is loading and for [accessibility](https://developer.mozilla.org/en-US/docs/Web/Accessibility) reason. Kindly address this. :smile:

File: index.html [hunk 4]
```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
```
Comment (Kaghenimbale) [inline index.html:6]: - [ ]  Kindly I suggest you add a descriptive title into your <title> tag :thinking: eg:  "Product Landing Page". As instructed in the README provided. A descriptive title allows a user to easily identify what Web page they are using. It can be used to identify the Web page without requiring users to read or interpret page content. Please let's address this :+1:

File: index.html [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+          <li class="list">Terms</li>
+          <li class="list">Contact</li>
+        </ul>
+      </div>
+      <p>Copyright 2016, Original Trombones</p>
+    </footer>
+
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <script src="js/index.js"></script>
```
Comment (Kaghenimbale) [inline index.html:155]: Can you please remove these script tags? It might be better to you keep your code clean.

**Review by Kaghenimbale — COMMENTED** [review]
### Hi @Kaghenimbale,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 
- Descriptive commit history.  ✔️  
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_
Submit for review when you are done.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @Kaghenimbale when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (Nyame-Wolf) [reply index.html:6]: I suggest avoid using the word `can`. Even though you used the word please, still feels more like an order.
You could rephrase it to:
`Please  change the title of the page, As  instructed in the README`

File: index.html (hunk 1 above)
Comment (Nyame-Wolf) [reply index.html:47]: Kindly add a reason for requesting this change. it might be obvious for you but not for someone else.

File: index.html (hunk 2 above)
Comment (Nyame-Wolf) [reply index.html:63]: Kindly add a reason for requesting this change. it might be obvious for you but not for someone else.

File: index.html (hunk 3 above)
Comment (Nyame-Wolf) [reply index.html:79]: Kindly add a reason for requesting this change. it might be obvious for you but not for someone else.

**Review by Nyame-Wolf — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle: 

Hi @Kaghenimbale  :wave: 

Your project page is looking fantastic so far :star_struck: ! 

However, there is still some room for improvement, that can help you have better code and make your project professional :sunglasses: 



## Highlights :sparkles:

- Used polite language
- Used Github review feature well

Congrats on your work so far! :clap:  

## Required Changes ♻️

There are some issues that you still need to work on to go to the next project, but you are almost there! 
_For more information, check the comments under the review :arrow_down:_

## Optional suggestions :ballot_box_with_check: 

_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code :100:_

Happy coding and keep it up! :raised_hands: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
  Please, remember to tag me **@Nyame-Wolf**  in your question so I can receive the notification.

  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

  _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html (hunk 1 above)
Comment (Shedrack-Sunday) [reply index.html:47]: > Can you please add a text in the alt attribute? It might give the user an idea of the type the image that should be displayed when the image is loading.

- [ ] Just as the former reviewer pointed out in other sections of this review which has the same tune as this, please rephrase the starting point of your review to sound a little bit more friendly 😶‍🌫️ ...Using "Can you please add a text attribute " though with a plea of "Please", till sound authoritative with no warm turn. Remember that these reviews are guidelines to help you. I understand that accent and background might be a factor, but please heed to the advice given, and let's get that improved 😃 .

For example use:
`- [ ] I suggest you please add a text in the alt attribute. This is so because ...(continue with a "why")`

File: index.html (hunk 4 above)
Comment (Shedrack-Sunday) [reply index.html:6]: > Please can you change the title of the page, As instructed in the README

- [ ] Words are important factors in the way humans react. the previous reviewer was right. please rephrase your text. find thee suggestions 
- `Kindly change the title of the page, As instructed in the README provided.(include the link path as an option)..This is so because (conture with a "why")`
- `I suggest you change the title of the page to tally with that given on the README Template. Following such instructions will.......continue!`

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @Kaghenimbale   ,


CHANGES REQUIRED ♻️
Great work on making considerable progress in implementing the changes from the previous reviewer so far 👏
However, there are still a few which aren't been addressed yet.




Highlight
✅ Proper setup
✅ Descriptive commit messages


Required Changes ♻️

- [ ] There are still changes the previous reviewer pointed out in the way you should tune your sentences, which I strongly suggest you heed. Put yourself in the shoes of future persons who you might be reviewing, and understand that communication is an important factor in how people take feedback. 👏 

Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.


Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 2 above)
Comment (Kaghenimbale) [reply index.html:63]: - [ ] Great job adding the icon image :+1:   However as suggested above :upside_down_face: you could also add text in the alt attribute of the img tag, this might give to the user an idea of the image type you are displaying when the image is loading and for [accessibility](https://developer.mozilla.org/en-US/docs/Web/Accessibility) reason. Kindly address this. :smile:

**Review by amejid — APPROVED** [review]
Hi @Kaghenimbale, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Comments are polite and provide a reason :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
